### PR TITLE
fix build dependency

### PR DESCRIPTION
**Pull request by xquanluu**

### CODE REVIEW

File: Makefile.am [hunk 1]
```diff
@@ -84,8 +84,6 @@ BUILT_SOURCES=${srcdir}/deps/boost/stage/lib/libboost_thread.a ${srcdir}/deps/so
 
 clean-local:
 	cd ${srcdir}/deps/boost && ./bootstrap.sh && ./b2 clean
-	cd ${srcdir}/deps/sofia-sip && make clean
```
Comment (davehorton) [inline Makefile.am:87]: wondering why you deleted these lines?

Comment (xquanluu) [reply Makefile.am:87]: in CI check, there is no target for clean.

Comment (davehorton) [reply Makefile.am:87]: but I want clean-local to clean sofia-sip as well, and now it wont so I am still not following why this was necessary

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -15,7 +15,7 @@ jobs:
       - name: build dependencies
         run: |
           sudo apt-get update
-          sudo apt-get install -y libtool libtool-bin libcurl4-openssl-dev libpcap-dev libgoogle-perftools-dev
+          sudo apt-get install -y libtool libtool-bin libcurl4-openssl-dev libpcap-dev libunwind-dev libgoogle-perftools-dev
```
Comment (davehorton) [inline .github/workflows/ci.yml:18]: why is libunwind-dev required?  What library is depending on it?

**Review by davehorton — APPROVED** [review]

**Pull request merged**